samples/python/reactors/combustor.py

```diff
@@ -43,32 +46,34 @@
 combustor = ct.IdealGasReactor(gas)
 combustor.volume = 1.0
 
-# Create a reservoir for the exhaust
+# %%
+# Create a reservoir for the exhaust:
 exhaust = ct.Reservoir(gas)
 
+# %%
 # Use a variable mass flow rate to keep the residence time in the reactor
 # constant (residence_time = mass / mass_flow_rate). The mass flow rate function
 # can access variables defined in the calling scope, including state variables
-# of the Reactor object (combustor) itself.
-
-
+# of the `Reactor` object (combustor) itself.
```

Comment:
In some places you have two backticks for monospace/API links, here's there's a single one. I didn't notice a pattern between the cases, so just asking for a double check :smile:

Reply:
For ReST formatted documents, the single backtick utilizes the default Sphinx "role", which we have set to link to Python API objects, while double backticks is used for monospaced text. I double checked and I think all of the changes here are consistent with that usage.

Reply:
Thanks for double checking!